core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -1267,7 +1274,7 @@ protected void bitcoinSerializeToStream(OutputStream stream) throws IOException
         for (TransactionOutput out : outputs)
             out.bitcoinSerialize(stream);
         uint32ToByteStreamLE(lockTime, stream);
-        if(getVersionShort() >= 3 && getType() != Type.TRANSACTION_NORMAL) {
+        if(getVersionShort() >= SPECIAL_VERSION && getType() != Type.TRANSACTION_NORMAL) {
```

Comment:
You didn't mention this one, but I think `SPECIAL_VERSION` has to be used here as well instead of `MAX_STANDARD_VERSION`?

Reply:
Correct, I missed this one.